askama_parser/src/node.rs

```diff
@@ -510,6 +516,26 @@ impl<'a> Macro<'a> {
     }
 }
 
+fn check_end_name<'a>(
```

Comment:
This shouldn't move in the second commit.

Reply:
You told me to put it just after the highest user. Since the highest user changed, I moved it. I'm gonna revert this move then.

Reply:
You're strictly correct -- I'd have added it in the first commit in the place where it would be after the second user introduced it, balancing the concerns of (a) clean commits that don't introduce unnecessary changes within a PR, and (b) a clean ordering at the end of the PR.